packages/ai-history/src/browser/ai-history-widget.tsx

```diff
@@ -41,11 +51,38 @@ export class AIHistoryView extends ReactWidget {
         this.title.iconClass = codicon('history');
     }
 
+    protected get state(): AIHistoryView.State {
+        return this._state;
+    }
+
+    protected set state(state: AIHistoryView.State) {
+        this._state = state;
+        this.onStateChangedEmitter.fire(this._state);
+    }
+
+    get onStateChanged(): Event<AIHistoryView.State> {
```

Comment:
usually this is done by:
readonly onStateChanged = this.onStateChangedEmitter.event;

Reply:
who is listening to this outside the widget?